### PR TITLE
Adds the basic bones code of conduct

### PR DESCRIPTION
**Pull request by Cyberboss**
## About The Pull Request

A sign for the bus driver to tap when people are out of line.

## Why It's Good For The ~~Game~~ Repo

Removes this page from the book of grudges
image

Also something something recent events on this station have left me very disappointed, okay?

@tgstation/commit-access

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
```
Comment (MrStonedOne) [inline CODE_OF_CONDUCT.md:13]:
```suggestion
inclusive, and healthy community.
```

Diverse is a loaded term that implies discrimination.

**Review by Watermelon914 — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
```
Comment (MrStonedOne) [inline CODE_OF_CONDUCT.md:24]:
```suggestion

```

"affected by our mistakes" is overlay vague and broad and gives power to bullies to claim to be affected by minute mistakes.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```
Comment (MrStonedOne) [inline CODE_OF_CONDUCT.md:37]:
```suggestion
```

This is a hobby, not a job, "professional setting" is overloaded

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
```
Comment (MrStonedOne) [inline CODE_OF_CONDUCT.md:49]:
```suggestion
decisions.
```

They should always communicate reasons for moderation decisions.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+## Enforcement Guidelines
+
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
```
Comment (MrStonedOne) [inline CODE_OF_CONDUCT.md:77]:
```suggestion
unwelcome in the community.
```

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
@tgstation-server, you looking to fill that spot in the book of grudges?

**Comment by MrStonedOne** [comment]
didn't you code the primary logic for auto-tagging?

**Comment by CRITAWAKETS** [comment]
Kinda always felt the contributor covenant specifying things to not discriminate against instead of just "dont discriminate" just feels iffy since its way too long for the message it conveys while also opening the door to some asshat finding a loophole.

**Comment by Cyberboss** [comment]
> 
> 
> didn't you code the primary logic for auto-tagging?

https://user-images.githubusercontent.com/8171642/140167429-c06d2a4d-5949-429d-8e2a-fa633cb48647.mov

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[#coding-general in the /tg/station 13 Discord Guild](https://discord.gg/tg13).
```
Comment (Iamgoofball) [inline CODE_OF_CONDUCT.md:59]: please don't call it a guild

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+All complaints will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
+## Enforcement Guidelines
```
Comment (Iamgoofball) [inline CODE_OF_CONDUCT.md:65]: given the community culture, we should have some form of formalized appeals process in here

Comment (Iamgoofball) [reply CODE_OF_CONDUCT.md:65]: currently it's literally just "PM ninjanomnom or oranges in discord with your apologies/explanations"

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
```
Comment (Iamgoofball) [inline CODE_OF_CONDUCT.md:30]: we should add a qualifier here about slurs, i know it's implied via the wording here but it's best to be clear

**Comment by Krysonism** [comment]
This is blob of soulless corporate boilerplate that sounds like it was drafted by a PR firm.

Inauthentic, meaningless and alienating.

**Comment by msgerbs** [comment]
What about this is alienating?

**Comment by optimumtact** [comment]
we don't need one, common sense is more than suitable.

**Pull request closed without merging**